solr/core/src/java/org/apache/solr/cloud/api/collections/CreateCollectionCmd.java

```diff
@@ -702,13 +700,6 @@ public static void createCollectionZkNode(
               }
             }
 
-            // if the config name wasn't passed in, use the default
-            if (!collectionProps.containsKey(ZkController.CONFIGNAME_PROP))
-              collectionProps.put(ZkController.CONFIGNAME_PROP, defaultConfigName);
```

Comment:
you removed this logic, thus ignored the config name specified in the request.  Surely we want to continue to do that?

Reply:
I *think* that what I did was move this up to line 692, and removed the `if` conditional.   This code path is only entered if you have a `solr.configset.bootstrap.confdir` set, so then we get the default either from the `solr.configset.bootstrap.config.name` or if that is missing then the `collection` variable...